### ISSUE TITLE
problem with require

### ISSUE THREAD

**Issue by jvanvliet17** [18F/glossary#45]
For some reason, I don't manage to successfully run the following line: `var Glossary = require('glossary-panel');`

After running the code and looking at the console, I get the following error: 
`Uncaught ReferenceError: require is not defined`

When I instead try to use the whole path, I get the following error:
`Not allowed to load local resource: _nameofwholepath_`

Does anyone know by any chance why this is caused and how this can be solved.

**Comment by afeld** [comment]
Yeah, those instructions assume you are compiling your front end assets with Node.js, with Webpack or Browserify or something like that. Would be useful to provide a downloadable version that is frontend-ready.

**Comment by jvanvliet17** [comment]
@afeld  Thanks for pointing that out. I downloaded Browserify and subsequently via Browserify I made an attemp to creat a bundle.js, however then I get an error that `Error: Cannot find module 'terms'`
I put all the objects in a json file, however that didn't work
